lib/crc-model-visitor.js

```diff
@@ -3,31 +3,54 @@ const _ = require('lodash');
 
 (function () {
 
+    const defaultDeclarators = [
+        'FunctionDeclaration',
+        //'FunctionExpression',
+        //'ArrowFunctionExpression',
+        'ClassDeclaration',
+        'ClassExpression',
+        'VariableDeclaration',
+        'VariableDeclarator'
+    ];
+
+    let _customDeclarators = defaultDeclarators;
+
     /**
      * Collects information needed to construct an overall CRC Model
      * by "visiting" the abstract syntax tree.
      */
     class CrcModelVisitor {
 
         /**
+         *
+         * @static
+         * @member {CrcModelVisitor}
+         * @property {Array<string>} defaultDeclarators - The default set of node types to be used to identify classes.
+         *
+         */
+        static get defaultDeclarators() {
```

Comment:
This should be `read-only`. I'm evaluating the "official" way to implement `get` only access.